### PR TITLE
Updated Links On TwitterMediaClient.scala

### PR DESCRIPTION
**Pull request by lapmid**

### CODE REVIEW

**Comment by DanielaSfregola** [comment]
This is a partial fix for #149

File: src/test/scala/com/danielasfregola/twitter4s/http/clients/rest/media/TwitterMediaClientSpec.scala [hunk 1]
```diff
@@ -1,42 +1,167 @@
-package com.danielasfregola.twitter4s.http.clients.rest.media
```
Comment (DanielaSfregola) [inline src/test/scala/com/danielasfregola/twitter4s/http/clients/rest/media/TwitterMediaClientSpec.scala]: This file needs to be reverted

Comment (lapmid) [reply src/test/scala/com/danielasfregola/twitter4s/http/clients/rest/media/TwitterMediaClientSpec.scala]: which one, On which i have done a commit recently ??

Comment (DanielaSfregola) [reply src/test/scala/com/danielasfregola/twitter4s/http/clients/rest/media/TwitterMediaClientSpec.scala]: You PR is trying to also change the file com/danielasfregola/twitter4s/http/clients/rest/media/TwitterMediaClientSpec.scala ... which is wrong :)

Comment (lapmid) [reply src/test/scala/com/danielasfregola/twitter4s/http/clients/rest/media/TwitterMediaClientSpec.scala]: TwitterMediaClientSpec.scala  m unable to find this file please help

**Comment by lapmid** [comment]
Okay

**Comment by DanielaSfregola** [comment]
Hi @lapmid ,
thanks for your contribution!

LGTM

**Pull request merged**